multicore_runtime/legacy_e2e_support.py

```diff
@@ -86,18 +86,23 @@ def legacy_app_for_script(script):
     The application as extracted from the script, or None.
   """
   with MockedWsgiHandler.python_25_app_lock:
-    modname, _ = script.rsplit('.', 1)
+    try:
+      modname = script.rsplit('.', 1)[0]
```

Comment:
after this change, we don't need the "try except", right? unless you also want to catch the case that script is None